### PR TITLE
Use migration v12

### PR DESCRIPTION
**Pull request by ben-kaufman**

### CODE REVIEW

File: test/0.1.1-rc.12/domain.spec.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ describe('Domain Layer', () => {
   let accounts;
   const orgName = getOrgName();
   const tokenName = orgName + ' Token';
-  const tokenSymbol = orgName[0] + orgName.split(' ')[1][0] + 'T';
+  const tokenSymbol = 'MY';
```
Comment (jellegerbrandy) [inline test/0.1.1-rc.12/domain.spec.ts:31]: why this change?

Comment (ben-kaufman) [reply test/0.1.1-rc.12/domain.spec.ts:31]: Because Jordan changed the DAO name and the token name and symbol to be constant
